canarytokens/settings.py

```diff
@@ -8,7 +8,7 @@
 
 
 class SwitchboardSettings(BaseSettings):
-    PUBLIC_DOMAIN: str
+    PUBLIC_DOMAIN: str = ""
```

Comment:
I'd consider leaving `PUBLIC_DOMAIN` and `WG_PRIVATE_KEY_SEED` as values that must be specified unless we've thoroughly tested that them being blank doesn't break things...

Reply:
Yessir! I think thats a good idea